wiki/articles/maven-integration.md

```diff
@@ -1,29 +1,25 @@
 ---
 title: Maven integration
 ---
-Instead of using Gradle to setup your libGDX project, you can also use Maven. However, **this is _not_ recommended!** When issues arise, getting support will most certainly prove very difficult!
+*last update: 13/03-2024 (DDMMYYYY)*
 
-The article below may be outdated. Use it at your own risk!
-{: .notice--warning}
+Instead of using Gradle to setup your libGDX project, you can also use Maven. However, this is not as well supported!
 
 ## Introduction
 
-libGDX is currently published to the SonaType snapshot repository. This will continue to happen until the Maven integration is complete. The repository only contains the latest snapshot release for now. Eventually we will publish release builds to SonaType as well.
-
 Add the following to your settings.xml or your project's pom.xml:
 
 ```xml
-  <repositories>
-    <repository>
-      <id>gdx-nightlies</id>
-      <url>https://oss.sonatype.org/content/repositories/snapshots/</url>
-    </repository>
-  </repositories>
+  <dependency>
```

Comment:
I don't know, what this is supposed to do. This is certainly not enought to setup libGDX.
Since the old paragraph was about setting up the snapshot repositories, I think this can be cut instead?

Reply:
It actually is, for desktop at least. Although I did miss some, my bad. 

```xml
         <dependency>
            <groupId>com.badlogicgames.gdx</groupId>
            <artifactId>gdx-backend-lwjgl3</artifactId>
            <version>${gdx.version}</version>
        </dependency>
        <dependency>
            <groupId>com.badlogicgames.gdx</groupId>
            <artifactId>gdx-platform</artifactId>
            <version>${gdx.version}</version>
            <classifier>natives-desktop</classifier>
        </dependency>
        ```
        
        Insert that in your maven pom.xml, init the lwjgl app with yours and you're good to go.